skillserver/echo.go

```diff
@@ -185,6 +186,14 @@ type EchoSession struct {
 	} `json:"user"`
 }
 
+type EchoContext struct {
+	System struct {
+		Device struct {
+			DeviceId string `json:"deviceId,omitempty"`
```

Comment:
It would be great to also add the supportedInterfaces property for anyone that wants to build audio or video skills.

Reply:
Good catch. We could always add it later if @sholzmayer doesn't want to make the change